src/protagonist/Test.Helpers/Integration/DatabaseTestDataPopulation.cs

```diff
@@ -207,4 +210,34 @@ public static class DatabaseTestDataPopulation
         asset.Result.Entity.WithTestThumbnailMetadata(metadataValue);
         return asset;
     }
+
+    public static ValueTask<EntityEntry<Asset>> WithTestDeliveryChannel(
+        this ValueTask<EntityEntry<Asset>> asset,
+        string deliveryChannel,
+        int? policyId = null)
+    {
+        if (!AssetDeliveryChannels.IsValidChannel(deliveryChannel))
+        {
+            throw new ArgumentOutOfRangeException(nameof(deliveryChannel), deliveryChannel,
+                $"Acceptable delivery-channels are: {AssetDeliveryChannels.AllString}");
+        }
+
+        asset.Result.Entity.ImageDeliveryChannels.Add(new ImageDeliveryChannel()
+        {
+            Channel = deliveryChannel,
+            DeliveryChannelPolicyId = policyId ?? deliveryChannel switch
+            {
+                AssetDeliveryChannels.Image => KnownDeliveryChannelPolicies.ImageDefault,
+                AssetDeliveryChannels.Thumbnails => KnownDeliveryChannelPolicies.ThumbsDefault,
+                AssetDeliveryChannels.Timebased => MIMEHelper.IsVideo(asset.Result.Entity.Origin)
+                    ? KnownDeliveryChannelPolicies.AvDefaultVideo
+                    : KnownDeliveryChannelPolicies.AvDefaultAudio,
+                AssetDeliveryChannels.File => KnownDeliveryChannelPolicies.FileNone,
+                _ => throw new ArgumentOutOfRangeException(nameof(deliveryChannel), deliveryChannel,

```

Comment:
As above - is this required? Or should it be controllable via a param to this method?